x-pack/plugins/enterprise_search/public/applications/app_search/components/result/result_actions.tsx

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { EuiButtonIcon, EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+
+import { ResultAction } from './types';
+
+interface Props {
+  actions: ResultAction[];
+}
+
+export const ResultActions: React.FC<Props> = ({ actions }) => {
```

Comment:
Good call on moving this out to it's own component, this simplifies things greatly.